api/src/test/java/uk/gov/hmcts/payment/api/componenttests/CreditAccountPaymentControllerTest.java

```diff
@@ -324,15 +327,13 @@ public void retrievePaymentStatusesTest() throws Exception {
     }
 
     @Test
-    public void validateCreateCreditAccountPayment_withoutCcdCaseNumberAndCaseReferenceTest() throws Exception {
-        CreditAccountPaymentRequest request = objectMapper.readValue(jsonRequestWithoutCcdCaseRefAndCaseRef().getBytes(), CreditAccountPaymentRequest.class);
+    public void validateCreateCreditAccountPayment_InvalidCcdCaseNumberTest() throws Exception {
+        CreditAccountPaymentRequest request = objectMapper.readValue(jsonRequestWithInvalidCcdCaseRef().getBytes(), CreditAccountPaymentRequest.class);
 
         MvcResult res = restActions
             .post("/credit-account-payments", request)
             .andExpect(status().isUnprocessableEntity())
```

Comment:
Would be good to check for the Reponse Message as well please as it will help in Support and Debugging...